src/d3a/d3a_core/sim_results/stats.py

```diff
@@ -67,11 +67,12 @@ def __init__(self, is_spot_market=True):
     @classmethod
     def _store_bought_trade(cls, result_dict, trade):
         # Division by 100 to convert cents to Euros
+        fee_price = trade.fee_price / 100. if trade.fee_price is not None else 0.
         result_dict['bought'] += trade.offer.energy
-        result_dict['spent'] += trade.offer.price / 100.
+        result_dict['spent'] += trade.offer.price / 100. - fee_price
```

Comment:
Can you please explain why you subtract the fees here and add them later again?

Reply:
If you look into the attached pdf of ticket "bought" --> "Trade Cost" , it excluded the grid fee. We can discuss offline too if you need further explanation. Thanks!

Reply:
Ah, i see, so you are omitting the fees fro the device bills, but the accumulated Trades include them. Ok then. Thanks!